### ISSUE TITLE
documentation/ofxAssimpModelLoader/ incongruity

### ISSUE THREAD

**Issue by zeffii** [openframeworks/ofSite#325]
verbatim from [/ofxAssimpModelLoader.html](http://openframeworks.cc/documentation/ofxAssimpModelLoader/ofxAssimpModelLoader.html)

> In order to load a file to memory, you just need to declare a new variable of type ofAssimpModelLoader and use the function loadModel():
```C++
ofxAssimpModelLoader model;
model.loadModel("file_name.3DS");
```

I'm fairly rusty with the C++ but that seems like a typo in the leading text. It would help tremendously if there was a link to an example project. I realize you're all busy people, I'd do it myself but I'm just starting with openframeworks ( really nice project btw. wow.)

Somewhere at the top of  that page it could at least say something like: "To see an example of ofxAssimpleModelLoader checkout _examples/addons/assimpExample_"

**Comment by kylemcdonald** [comment]
good catch, this seems to appear a few places in the documentation.

for an example see `openFrameworks/examples/addons/assimpExample`

**Comment by zeffii** [comment]
go with the concise :)

**Comment by arturoc** [comment]
Closed by #362
